Mjml.Net/Includes/Filesystem/FilesystemPathResolver.cs

```diff
@@ -0,0 +1,37 @@
+﻿using Mjml.Net.Components;
+
+namespace Mjml.Net.Includes.Filesystem;
+
+/// <summary>
+///     Resolves full path for <see cref="IncludedFileInfo" />
+/// </summary>
+public class FilesystemPathResolver : IMjIncludePathResolver
```

Comment:
The abstraction feels a little bit weird, because why would you combine a FileSystemPathResolver with any non file based system?

Reply:
Because you can have any kind of path resolvers in theory. I think `Path.Combine` is not a silver bullet here. And moreover, you could not be satisfied by built-in one, so I decided to give developer free of will here. Maybe someone will prefer to write their own, that is not based on `Path.Combine`.